### PR TITLE
Add i18n-support

### PR DESCRIPTION
**Pull request by rixx**

### CODE REVIEW

File: pretalx_lower_thirds/forms.py [hunk 1]
```diff
@@ -1,23 +1,21 @@
 from django import forms
```
Comment (Kunsi) [inline pretalx_lower_thirds/forms.py:1]: That import is now unused, please remove it.

File: pretalx_lower_thirds/forms.py [hunk 2]
```diff
@@ -1,23 +1,21 @@
...
 
 
-class LowerThirdsSettingsForm(HierarkeyForm):
-    lower_thirds_no_talk_info = forms.CharField(
+class LowerThirdsSettingsForm(I18nFormMixin, HierarkeyForm):
+    lower_thirds_no_talk_info = I18nFormField(
         help_text=_(
-            "Will be shown as talk title if there's currently no talk "
-            "running."
+            "Will be shown as talk title if there's currently no talk " "running."
```
Comment (Kunsi) [inline pretalx_lower_thirds/forms.py:10]: That indentation was there to not have too long lines. Feel free to remove it, but please also remove the unneeded quotes then :)

**Review by Kunsi — CHANGES_REQUESTED** [review]

File: pretalx_lower_thirds/signals.py [hunk 3]
```diff
@@ -1,8 +1,19 @@
...
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import gettext_noop, ugettext_lazy as _
+from i18nfield.strings import LazyI18nString
+from pretalx.common.models.settings import hierarkey
 from pretalx.orga.signals import nav_event_settings
 
+hierarkey.add_default(
+    "lower_thirds_no_talk_info",
+    LazyI18nString.from_gettext(
+        gettext_noop("Sorry, there's currently no talk running")
```
Comment (Kunsi) [inline pretalx_lower_thirds/signals.py:11]: Does this also fix #2?

File: pretalx_lower_thirds/forms.py (hunk 2 above)
Comment (rixx) [reply pretalx_lower_thirds/forms.py:10]: Sorry, I committed auto-formatter changes – I habitually run `black` before committing.

File: pretalx_lower_thirds/signals.py (hunk 3 above)
Comment (rixx) [reply pretalx_lower_thirds/signals.py:11]: Sure does! The default value will be returned by `hierarkey` for every implicit null value.

**Pull request merged**